msgpack/_packer.pyx

```diff
@@ -56,6 +56,13 @@ cdef class Packer(object):
         Convert unicode to bytes with this encoding. (default: 'utf-8')
     :param str unicode_errors:
         Error handler for encoding unicode. (default: 'strict')
+    :param bool precise_mode:
+        If set to true, types will be checked to be exact. Derived classes
+        from serializeable types will not be serialized and will be
+        treated as unsupported type and forwarded to default.
```

Comment:
I don't like `precise_mode` name since it doesn't mention about types.
What about `strict_types`?